### ISSUE TITLE
Version links between operators

### ISSUE THREAD

**Issue by nightkr** [stackabletech/documentation#302]
Split off from #299.

We basically have two possible approaches here, I believe:

1. Add version tags for cross-operator links
2. Unify versioning across the platform and convince Antora to default to keep versions the same across operators

1 has less immediate impact, but we've been discussing doing 2 elsewhere already, and I believe that would be more maintainable over time (as well as being easier for end users to understand).

**Comment by fhennig** [comment]
:+1: 

In favor of 2.

**Comment by teozkr** [comment]
Agreed.

**Comment by fhennig** [comment]
I was interested, so I researched what would be the way to go. I think a distributed component would be the way: https://docs.antora.org/antora/latest/distributed-component-version/ . All operators would become part of the same logical component (the platform) and be versioned as one. All links would then be within the same component & version by default.

I've made a ticket with my thoughts https://github.com/stackabletech/documentation/issues/306
